libpod/boltdb_state.go

```diff
@@ -2347,11 +2347,19 @@ func (s *BoltState) AddPod(pod *Pod) error {
 		// Check if we already have something with the given ID and name
 		idExist := idsBkt.Get(podID)
 		if idExist != nil {
-			return errors.Wrapf(define.ErrPodExists, "ID %s is in use", pod.ID())
+			err = define.ErrPodExists
+			if allPodsBkt.Get(idExist) == nil {
+				err = define.ErrCtrExists
+			}
+			return errors.Wrapf(err, "ID \"%s\" is in use", pod.ID())
```

Comment:
For reference, `%q` produces the same output as `\"%s\"` - but since they're identical, there's no real need to change it.